### PR TITLE
[KOGITO-5252] Performance testing

### PR DESCRIPTION
**Pull request by fjtirado**
Merge with https://github.com/kiegroup/kogito-runtimes/pull/1420

For performance testing check https://github.com/kiegroup/kogito-examples/pull/801

### CODE REVIEW

File: process-performance-quarkus/pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-reactive-messaging-kafka</artifactId>
+    </dependency>
+<!--     <dependency> -->
+<!--   <groupId>org.kie.kogito</groupId> -->
+<!--   <artifactId>kogito-addons-quarkus-cloudevents</artifactId> -->
+<!-- </dependency> -->
```
Comment (ricardozanini) [inline process-performance-quarkus/pom.xml]: Leftovers?

**Review by ricardozanini — COMMENTED** [review]
Looks fine! You might consider improving the README a little bit more to add more context and properly configure the project. I understand that it seems for our internal usage/experiments, but users might find this useful to run their own benchmarks.

File: process-performance-quarkus/pom.xml (hunk 1 above)
Comment (fjtirado) [reply process-performance-quarkus/pom.xml]: just there to quickly switch from the two different kafka implementations, Ill kept it for a while, but docummenting that

**Comment by fjtirado** [comment]
> Looks fine! You might consider improving the README a little bit more to add more context and properly configure the project. I understand that it seems for our internal usage/experiments, but users might find this useful to run their own benchmarks.

I agree, actually I should open this as draft ;)

**Comment by fjtirado** [comment]
jenkins retest this

**Comment by fjtirado** [comment]
Hi @ricardozanini @tiagodolphine 
@tiagodolphine mentions that maybe this code should not go to examples repositories (I agree it is not really an example, although it contain bpmn examples ;))
@ricardozanini argues that this might be useful to customer (hence he votes for enhancing readme files and keep the code here). 
What should I do?

**Comment by ricardozanini** [comment]
@fjtirado @tiagodolphine @MarianMacik @sutaakar @radtriste I think it would be useful to have a project (`kogito-qe`?) for tests tooling/performance/benchmarks. This way we can add things there that are related to this extremely important work. It will be easy to share with the community and publish our benchmarks. wdyt?

**Comment by radtriste** [comment]
@ricardozanini Indeed, this is not as itself an example, even if it could be useful to the user.
I think we should have a `kogito-testing` repository for this kind of testing.

**Comment by fjtirado** [comment]
@radtriste I think is a great idea to have a different repo to include this kind of stuff, how do we create one?

**Comment by sutaakar** [comment]
Fine for me, but lets wait for Marian. He is the main person for performance testing. Maybe there is already some repository.

**Comment by fjtirado** [comment]
Im going to use this PR to fix an issue in optaplanner example that is preventing green light for https://github.com/kiegroup/kogito-runtimes/pull/1420, I opened https://github.com/kiegroup/kogito-examples/pull/801 to follow up discussion on performance testing.

**Comment by cristianonicolai** [comment]
Jenkins rerun examples tests

**Review by cristianonicolai — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: process-kafka-multi-quarkus/pom.xml [hunk 2]
```diff
@@ -75,6 +59,10 @@
       <artifactId>cloudevents-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-events-smallrye</artifactId>
+</dependency>
```
Comment (danielezonca) [inline process-kafka-multi-quarkus/pom.xml:65]: Fix formatting

**Review by danielezonca — APPROVED** [review]

**Pull request merged**